dna_analyse/DNA_Analyse.py

```diff
@@ -47,8 +45,30 @@ def hole_dna_sequenz():
 
 
 seq = hole_dna_sequenz()
-# Hier sollten jetzt die nötigen Berechnungen durchgeführt werden.
 
-# Und hier erfolgt dann die Ausgabe.
+Anzahl_G = seq.count('G') + seq.count('g')
+Anzahl_A = seq.count('A') + seq.count('a')
+Anzahl_T = seq.count('T') + seq.count('t')
+Anzahl_C = seq.count('C') + seq.count('c')
```

Comment:
wie schon in den Übungen besprochen könnte man stattdessen einfach mit `seq.upper()` arbeiten, um den Code zu vereinfachen.